### PR TITLE
Postfunction

### PR DESCRIPTION
**Pull request by jcalvarezu**

### CODE REVIEW

File: exercise-services/.classpath [hunk 1]
```diff
@@ -6,11 +6,6 @@
 			<attribute name="maven.pomderived" value="true"/>
```
Comment (joserv2812) [inline exercise-services/.classpath]: read about .gitignore and add this in there

File: exercise-services/.project [hunk 2]
```diff
@@ -21,12 +21,12 @@
 			</arguments>
```
Comment (joserv2812) [inline exercise-services/.project]: .gitignore

File: exercise-services/src/main/java/com/pluralsight/ActivityResource.java [hunk 3]
```diff
@@ -18,6 +21,39 @@
   
   private ActivityRepository activityRepository = new ActivityRepositoryStub();
   
+  @POST
+  @Path("activity")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+  public Activity createActivity(Activity activity){
+	
+	System.out.println(activity.getDescription());
```
Comment (joserv2812) [inline exercise-services/src/main/java/com/pluralsight/ActivityResource.java]: delete System.out.println();

File: exercise-services/src/main/java/com/pluralsight/ActivityResource.java [hunk 4]
```diff
@@ -18,6 +21,39 @@
...
+    
+  }
+  
+  @POST
+  @Path("activity")
+  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+  @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+  public Activity createActivityParams(MultivaluedMap<String, String> formParams) {
+	
+	System.out.println(formParams.getFirst("description"));
```
Comment (joserv2812) [inline exercise-services/src/main/java/com/pluralsight/ActivityResource.java]: delete System.out.println();

File: exercise-services/src/main/java/com/pluralsight/ActivityResource.java [hunk 5]
```diff
@@ -18,6 +21,39 @@
...
+	
+	Activity activity = new Activity();
+	activity.setDescription(formParams.getFirst("description"));
+	activity.setDuration(Integer.parseInt(formParams.getFirst("duration")));
+	
+	activityRepository.create(activity);
+	
+	return activity;
+  }
+
```
Comment (joserv2812) [inline exercise-services/src/main/java/com/pluralsight/ActivityResource.java:50]: use 2 spaces for indentation

File: exercise-services/src/main/java/com/pluralsight/ActivityResource.java [hunk 6]
```diff
@@ -18,6 +21,33 @@
   
   private ActivityRepository activityRepository = new ActivityRepositoryStub();
   
+  @POST
+  @Path("activity")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+  public Activity createActivity(Activity activity){
+	
+	activityRepository.create(activity);
```
Comment (joserv2812) [inline exercise-services/src/main/java/com/pluralsight/ActivityResource.java:30]: fix indentation

**Comment by joserv2812** [comment]
why .gitignore is empty?

**Comment by joserv2812** [comment]
squash commits in one

**Comment by jcalvarezu** [comment]
All the commits are already together using squash. Ready to merge into develop.

**Pull request merged**